plone/recipe/codeanalysis/tests/test_flake8.py

```diff
@@ -8,9 +8,101 @@
 from tempfile import mkdtemp
 from unittest import TestCase
 
+VALID_CODE = '''\
+# -*- coding: utf-8 -*-
+def foobar():
+    """
+    A docstring with a "comment"
+    """
+    if ('foo' == 'bar'):
+        return 'foobar'
+
+    # A comment with "quote"
+    def bar(baz):
+        """a method "called" bar"""
+        return baz  # could also be "baz"
+
+    def foo(baz):
+        \'\'\'a method "called" foobar\'\'\'
```

Comment:
according to PEP 257, for consistency we should always use """triple double quotes""" around docstrings; I don't know if this should fail then, not because PEP 8, but because PEP 257.